web/pages/index.tsx

```diff
@@ -16,6 +16,10 @@ const Home = () => {
       <Head>
         <title>Styleguide – Variant</title>
         <link rel="icon" href={favicon} />
+        <link
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
```

Comment:
Luuurer på om det er bare bedre å bruke en svg-ikon her. Ikke så kritisk med styleguide men føles unødvendig å ta ned 170kb for et enkelt ikon

Reply:
Da blir det SVG Ikon, vi kan lage en PR på det 😉